src/Explorer/SplashScreen/FabricHome.tsx

```diff
@@ -157,17 +161,25 @@ export const FabricHomeScreen: React.FC<SplashScreenProps> = (props: SplashScree
 
   const title = "Build your database";
   return (
-    <div className={styles.homeContainer}>
-      <div className={styles.title} role="heading" aria-label={title}>
-        {title}
-      </div>
-      {getSplashScreenButtons()}
-      <div className={styles.footer}>
-        Need help?{" "}
-        <Link href="https://cosmos.azure.com/docs" target="_blank">
-          Learn more <img src={LinkIcon} alt="Learn more" />
-        </Link>
-      </div>
-    </div>
+    <>
+      <CosmosFluentProvider className={styles.homeContainer}>
+        <SampleDataImportDialog
+          open={openSampleDataImportDialog}
+          setOpen={setOpenSampleDataImportDialog}
+          explorer={props.explorer}
+          databaseName={userContext.fabricContext?.databaseName}
+        />
+        <div className={styles.title} role="heading" aria-label={title}>
+          {title}
+        </div>
+        {getSplashScreenButtons()}
+        <div className={styles.footer}>
+          Need help?{" "}
+          <Link href="https://cosmos.azure.com/docs" target="_blank">
```

Comment:
This link doesn't go anywhere ?

Reply:
Hhhm, that's a good question, it's not new code... where is it supposed to lead to?

Reply:
Maybe https://learn.microsoft.com/azure/cosmos-db/? But then we should have an aka.ms link for this!

Reply:
Oh, my bad, that's only for Fabric

Reply:
fixed 95c9230b936dc824c6b21d3f7ea3c14aac1c6984